Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -119,6 +119,31 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
         }
     }
 
+    private static func percentDecode(from: String?) throws -> String? {
+        if from == nil {
+            return nil
+        }
+        let decodedOptional = from?.removingPercentEncoding
```

Comment:
anther nit - you can one-line this, too, and save a variable:
`guard let decoded = from.removingPercentEncoding else {`